pkg/k8s/diverts/client.go

```diff
@@ -53,6 +54,8 @@ func NewForConfig(cfg *rest.Config) (*DivertV1Client, error) {
 }
 
 func GetClient(thisContext string) (*DivertV1Client, error) {
+	thisContext = okteto.UrlToContext(thisContext)
```

Comment:
what I mean is that `thisContext` shoul be always "", and use:
https://github.com/okteto/okteto/blob/master/pkg/k8s/client/client.go#L54

to point to our copy of kubeconfig in `$HOME/.okteto/contexts/kubeconfig`